tests/load/utils.py

```diff
@@ -302,6 +302,12 @@ def destinations_configs(
                 bucket_url=AZ_BUCKET,
                 extra_info="az-authorization",
             ),
+            DestinationTestConfiguration(
```

Comment:
This needs to be removed here and appended to the array at line 211, similar to how databricks is appended there.